code-studio/src/js/progress.js

```diff
@@ -33,6 +33,24 @@ progress.activityCssClass = function (result) {
 };
 
 /**
+ * See UsersHelper#activity_css_class.
+ * @param pageCompleted
+ * @return {string}
+ */
+progress.pageCompletedCssClass = function (pageCompleted) {
+  var PAGE_COMPLETED_PARTIAL = 1;
+  var PAGE_COMPLETED_YES = 2;
```

Comment:
Can these be mapped to existing activity constants?  Seems weird to introduce a new scale for dot colors.